homeassistant/components/sensor/translations/it.json

```diff
@@ -9,6 +9,10 @@
             "is_signal_strength": "Potenza del segnale attuale di {entity_name}",
             "is_temperature": "Temperatura attuale di {entity_name}",
             "is_timestamp": "Data e ora attuali di {entity_name}",
+            "is_current": "Corrente attuale di {entity_name}",
```

Comment:
Please remove the Italian translation. The translations will be handled by Lokalise.

Reply:
done